src/Views/Todos/Todos.js

```diff
@@ -0,0 +1,68 @@
+import './Todos.css';
+import React, { useEffect, useState } from 'react';
+// import { useHistory } from 'react-router-dom';
+import { fetchTodos, addTodo, toggleComplete } from '../../services/fetchtodos';
+
+
+export default function Todos() {
+  const [todo, setTodos] = useState([]);
+  const [makeTodo, setMakeTodo] = useState('');
+  const [error, setError] = useState('');
+//   const history = useHistory();
+
+  useEffect(() => {
+
+    const fetchData = async () => {
+      try {
+        const data = await fetchTodos();
+        setTodos(data);
+
+      } catch (error) {
+        setError(error.message);
+      }
+    };
+    fetchData();
+  }, []);
+
+  const handleClick = async (data) => {
+    try {
+      await toggleComplete(data);
+    } catch (e) {
+      setError('Ya broke it dummy.Click Testing');  
+    }
+  };
+
+//   setTodos((prevState) => ([…prevState, newTodo])
+
+  const handleSubmit = async () => { 
+      
+    try {  
+      const submit = await addTodo({ todo:makeTodo, complete:false });
+      setTodos((prevState) => [...prevState, submit]);
```

Comment:
This doesn't appear to be working. It could be due to using invalid syntax, such as todo and setTodos aren't equivalent for useState standards. Consider changing to todos and setTodos.